python/xorbits/deploy/kubernetes/core.py

```diff
@@ -196,6 +199,42 @@ async def request_worker(
     ) -> str:
         raise NotImplementedError
 
+    async def request_workers(
+        self, worker_num: int, timeout: Optional[int] = None
+    ) -> List[str]:
+        if worker_num <= 0:
+            raise ValueError("Please specify a `worker_num` that is greater than zero")
+        if timeout and timeout < 0:
+            raise ValueError("Please specify a `timeout` that is greater than zero")
+        start_time = time.time()
+        deployment = self._client_for_apps.read_namespaced_deployment(
+            name="xorbitsworker", namespace=self._k8s_namespace, _preload_content=False
```

Comment:
Don't use "xorbitsworker" directly. You can import it from XorbitsWorkersConfig.rc_name.